tests/Moq.Tests/ActionObserverFixture.cs

```diff
@@ -44,8 +44,8 @@ public void Void_method_call_with_coerced_arg()
 			public void Void_method_call_with_coerced_nullable_arg()
 			{
 				AssertReconstructable(
-					x => x.VoidWithNullableInt(42),
-					x => x.VoidWithNullableInt(42));
+					"x => x.VoidWithNullableInt(42)",
+					 x => x.VoidWithNullableInt(42));
```

Comment:
Need to find out why this change was necessary.

Reply:
Non-null literal for nullable parameters causes a `Convert`. We don't care about that, so a string comparison is fine.